.github/workflows/push-dispatch.yaml

```diff
@@ -0,0 +1,40 @@
+name: Push dispatch
+on:
+  push:
+    branches:
+      - main
+jobs:
+  changed-build:
+    runs-on: ubuntu-latest
+    name: Get changed builds
+    outputs:
+      changed-build: ${{ steps.changed-build.outputs.all_changed_files }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Get changed builds
+        id: changed-build
+        uses: tj-actions/changed-files@v28.0.0
+        with:
+          files: |
+            Jenkins/Dockerfile*
```

Comment:
The Dockerfile with no suffix is actually a symlink to another one Dockerfile.tf-torch-cpu. Would the wildcard ensure that only the actual file(s) that changed would trigger a rebuild and push of the docker image?

Reply:
I don't know the answer to this. Not familiar with this action and scenario. Probably would need some testing.

Reply:
Last I checked with Aaron he tested this works. I think we just need to merge and test. You don't have to use the generated dockerfile images until you've verified.